src/test/librbd/test_fixture.cc

```diff
@@ -98,6 +99,21 @@ int TestFixture::flatten(librbd::ImageCtx &ictx,
   return ictx.operations->flatten(prog_ctx);
 }
 
+int TestFixture::resize(librbd::ImageCtx *ictx, uint64_t size){
+  librbd::NoOpProgressContext prog_ctx;
+  return ictx->operations->resize(size, true, prog_ctx);
+}
+
+int TestFixture::update_object_map(librbd::ImageCtx *ictx, ceph::BitVector<2> &object_map) {
```

Comment:
Nit: if this is only a helper method for a single test, it should probably live within the test file itself.

Reply:
OK.